src/HtmlPurifier.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I think not. Should we go with PSR/fig or do we have our own rule? /cc @gbirke 

https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1494
https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#declare-statements-namespace-and-use-declarations